frontend/src/app/Settings/Facility/Components/OrderingProvider.tsx

```diff
@@ -105,66 +105,94 @@ const OrderingProvider: React.FC<Props> = ({
           }
           name="NPI"
           required={isRequired}
-          value={provider.NPI || ""}
-          onChange={onChange}
-          onBlur={() => {
-            validateField("orderingProvider.NPI");
-          }}
+          value={formCurrentValues.orderingProvider.NPI ?? undefined}
+          registrationProps={register("orderingProvider.NPI", {
+            required: isRequired
+              ? "NPI should be a 10-digit numerical value (##########)"
+              : "",
+            validate: {
+              validNPI: (npi: string) =>
+                npi?.length
+                  ? isValidNPI(npi) ||
+                    "NPI should be a 10-digit numerical value (##########)"
+                  : true,
+            },
+          })}
           validationStatus={
-            errors["orderingProvider.NPI"] ? "error" : undefined
+            errors?.orderingProvider?.NPI?.type ? "error" : undefined
           }
-          errorMessage={errors["orderingProvider.NPI"]}
+          errorMessage={errors?.orderingProvider?.NPI?.message}
         />
         <TextInput
           label="Phone number"
           name="op-phone"
           required={isRequired}
-          value={provider.phone || ""}
-          onChange={onChange}
-          onBlur={() => {
-            validateField("orderingProvider.phone");
-          }}
+          value={formCurrentValues.orderingProvider.phone ?? undefined}
           validationStatus={
-            errors["orderingProvider.phone"] ? "error" : undefined
+            errors?.orderingProvider?.phone?.type ? "error" : undefined
           }
-          errorMessage={errors["orderingProvider.phone"]}
+          errorMessage={errors?.orderingProvider?.phone?.message}
+          registrationProps={register("orderingProvider.phone", {
+            required: isRequired ? "Ordering provider phone is required" : "",
+            validate: {
+              validPhone: (opPhone: string) =>
+                opPhone?.length
+                  ? phoneNumberIsValid(opPhone) ||
```

Comment:
similar note to the comment [here](https://github.com/CDCgov/prime-simplereport/pull/5983/files#r1235287974)